doc/classes/BitMap.xml

```diff
@@ -71,6 +71,12 @@
 			<argument index="1" name="epsilon" type="float" default="2.0">
 			</argument>
 			<description>
+        Creates an [code]Array[/code] of polygons covering a rectangular portion of the bitmap. Each polygon is described as a [code]PoolVector2Array[/code] of its vertices.
```

Comment:
Indentation should be done with tabs (like in the rest of the file), not spaces.